state-transition/core/state_processor_staking.go

```diff
@@ -67,6 +68,19 @@ func (sp *StateProcessor) processOperations(
 		}
 	}
 
+	if version.EqualsOrIsAfter(blk.GetForkVersion(), version.Electra()) {
+		// After Electra, validators can request withdrawals through execution requests which must be handled.
+		requests, err := blk.GetBody().GetExecutionRequests()
+		if err != nil {
+			return err
+		}
+		for _, withdrawal := range requests.Withdrawals {
```

Comment:
Note: while we don't handle deposits or attestation requests, we may want to log that they have happened just to have observability over them